src/ext/java/org/opentripplanner/ext/fares/impl/OrcaFareService.java

```diff
@@ -394,14 +396,20 @@ private float getWashingtonStateFerriesFare(
     // WSF is free in one direction on each route
     // If a fare is not found in the map, we can assume it's free.
     // Route long name is reversed for the reverse direction on a single WSF route
-    return (fares != null && fares.get(wsfFareType) != null) ? fares.get(wsfFareType) : 0;
+    return (fares != null && fares.get(wsfFareType) != null)
+      ? fares.get(wsfFareType)
+      : usDollars(0);
```

Comment:
Can this be the universal `zero` you tried to use elsewhere? (multiple instances)

Reply:
I've introduced `Money.ZERO_DOLLARS` in https://github.com/opentripplanner/OpenTripPlanner/commit/633f42818749d08232eed4f5453715c0a102978c